docs/blueprints/audit-containment-v2.md

```diff
@@ -0,0 +1,67 @@
+This blueprint outlines some tweaks to the containment mechanism to both give the node operator early warning that something is wrong while giving them more time to resolve any issues. At the same time we still must prevent the node from getting out of giving a response to any audit initiated.
+
+This blueprint builds on the design as outlined in the blueprint [audit-containment.md](/docs/blueprint/audit-containment.md).
+
+## Main Objectives
+
+The objectives from the original blueprint carry over to the new design.
+1. Nodes won’t be able to “escape” audits for specific data by refusing to send requested data or going offline right after receiving a request from a Satellite.
+2. The audit service should attempt to verify the nodes' originally requested data in addition to continuing normal audits for that node.
+3. ContainmentDB holds metadata required to verify contained nodes' originally requested data.
+
+But additional objectives should be added.
+4. Nodes with temporary slow downs or issues with responsiveness should get a chance to fix those issues before disqualification.
+5. Node operators should receive some form of warning that their node is having issues.
+6. While the node operator is given time to fix issues, the network should start protecting itself for the eventuality that the node may be disqualified later on.
+
+## Design
+
+The original blueprint describes how the verifier will handle different audit results as follows:
+
+> the following cases can occur:
+> - Passes audit ->  `pendingAudit` is deleted from the ContainmentDB  -> stats updated in overlaycache/satelliteDB and `contained` field set to false
+> - Fails audit -> `pendingAudit` is deleted from the ContainmentDB -> stats updated in overlaycache/satelliteDB and `contained` field set to false
+> - Refuses to respond again -> the `pendingAudit`'s `reverifyCount` is incremented in the ContainmentDB ->
+>     - -> If that count exceeds the reverify limit, then the node’s stats will be updated to reflect an audit failure and the node will be removed from the ContainmentDB and `contained` field will be set to false.
+>     - -> If the count does not exceed the reverify limit, the `pendingAudit` will remain in the ContainmentDB.
+
+In this blueprint I suggest only a small addition to the very last scenario, where the node doesn't respond again, but the reverify limit hasn't been exceeded. In addition to leaving the `pendingAudit` in the ContainmentDB, the node's stats should be updated to reflect an unknown audit failure. Because these unkown audit failures will occur with every reverification, the node will be suspended before it is disqualified.
+
+With suspension happening before disqualification and the pieces marked as unhealthy already, we can now afford to raise the maxReverifyCount (suggestion: to 10) to allow the node operator more time to resolve any issues with their setup.
```

Comment:
This is a conflict with the goal that containment mode has. Containment mode should make sure nodes that just restart at the wrong moment don't get a penalty. Increasing the maxReverifyCount is not needed for that. The only reason to increase it would be to abuse containment mode beyond just catching nodes that restart at the wrong moment.